frontend/src/Pages/FrontScreen.js

```diff
@@ -27,18 +31,21 @@ class FrontScreen extends React.Component {
     }
   }
 
-  authenticate() {
+  authenticate = async() => {
     const microapps = window.microapps;
     const request = { nonce: 'nonce typically generated server-side' };
     microapps.getIdentity(request).then(response => {
       /* response: Concatenation of '.' separated base64 encoded JSON strings,
        * of which string at index 1 (0-based indexing) is the user identity.
        *
-       * Line 29: Decoding the base64 string at index 1 and parsing it as JSON
+       * Line below: Decoding the base64 string at index 1 and parsing it as JSON
        */
       const decoded = JSON.parse(atob(response.split('.')[1])); 
       console.log('GetIdentity response: ', decoded);
       this.props.auth(decoded);
+      this.setState({
+        "auth": true
+      });
```

Comment:
I don't see any use of this new state in the component. If it's not being used, consider getting rid of it.